email-alert-api/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.6.3
+
+RUN apt-get update -qq && apt-get upgrade -y && apt-get install -y build-essential vim
```

Comment:
I don't think we have build-essential in any of these any more since: https://github.com/alphagov/govuk-docker/pull/11